pages.ru/android/am.md

```diff
@@ -0,0 +1,20 @@
+# am
+
+> Android activity manager.
```

Comment:
Translate this line too.

Tip: Batch all my suggestions and make a single commit from `Files changed` tab.